VisualStudio.gitignore

```diff
@@ -80,6 +80,8 @@ ipch/
 *.opensdf
 *.sdf
 *.cachefile
+*.VC.db
+*.VC.VC.opendb
```

Comment:
Confirmation. `*.VC.db` and `*.VC.opendb` should be correct. Also, we can remove the `*.opensdf` entry I added. @vladon can you add that to this PR?